packages/eslint-plugin-pf-codemods/lib/rules/wizard-rename-hasBodyPadding.js

```diff
@@ -1,10 +1,42 @@
-const { renameProp } = require('../helpers');
+const { getPackageImports } = require('../helpers');
 
 // https://github.com/patternfly/patternfly-react/pull/4136
 module.exports = {
-  create: renameProp(
-    ['Wizard', 'WizardBody', 'WizardToggle'],
-    {'hasBodyPadding': 'hasNoBodyPadding'},
-    node => `hasBodyPadding prop has been removed for ${node.name.name}. Use hasNoBodyPadding instead`
-  )
-};
\ No newline at end of file
+  create: function(context) {
+    const imports = getPackageImports(context, '@patternfly/react-core')
+      .filter(specifier => specifier.imported.name === 'Wizard');
```

Comment:
I know the PR body didn't mention this, but `WizardBody` and `WizardToggle` also had this prop removed. Change to `['Wizard', 'WizardBody', 'WizardToggle'].includes(specifier.imported.name);`